### PR TITLE
Standardise style and use of backticks in prezi3

### PR DESCRIPTION
**Pull request by tomcrane**

### CODE REVIEW

**Comment by tomcrane** [comment]
I have increased the line spacing by .1em, which prevents `code highlights` on adjacent lines merging in to each other.

**Comment by tomcrane** [comment]
Code highlight colour f2f2f2

File: source/api/presentation/3.0/index.md [hunk 1]
```diff
@@ -1198,15 +1199,15 @@ The Canvas represents an individual page or view and acts as a central point for
...
+Parts of Canvases _MAY_ also be identified by appending a fragment to the Canvas's URI, and these parts are still considered to be Canvases and have a `type` of Canvas. Rectangular spatial parts of Canvases _MAY_ also be described by appending an `xywh=` fragment to the end of the Canvas's URI. Similarly, temporal parts of Canvases _MAY_ be described by appending a `t=` fragment to the end of the Canvas's URI. Spatial and temporal fragments _MAY_ be combined, using an `&` character between them, and the temporal dimension _SHOULD_ come first. It is an error to select a region using a dimension that is not defined by the Canvas, such as a temporal region of a Canvas that only has height and width dimensions.
```
Comment (azaroth42) [inline source/api/presentation/3.0/index.md]: I think: `type` value of `"Canvas"`  ?

Comment (azaroth42) [reply source/api/presentation/3.0/index.md]: Alternative:  and have the appropriate `type`.   (to be silent about the value which is obvious)

**Comment by tomcrane** [comment]
fixes #1533, #1538

File: source/api/presentation/3.0/index.md [hunk 2]
```diff
@@ -1357,11 +1359,11 @@ The content resource is linked in the `body` of the Annotation. The content reso
...
 
-If the content resource is an Image, and a IIIF Image service is available for it, then the URI _MAY_ be a complete URI to any particular representation made available, such as `https://example.org/image1/full/1000,/0/default.jpg`, but _MUST NOT_ be just the URI of the IIIF Image service. It _MUST_ have a `type` of "Image". Its media type _MAY_ be listed in `format`, and its height and width _MAY_ be given as integer values for `height` and `width` respectively. The image then _SHOULD_ have the service referenced from it.
+If the content resource is an Image, and a IIIF Image service is available for it, then the URI _MAY_ be a complete URI to any particular representation made available, such as `https://example.org/image1/full/1000,/0/default.jpg`, but _MUST NOT_ be just the URI of the IIIF Image service. It _MUST_ have a `type` of `Image`. Its media type _MAY_ be listed in `format`, and its height and width _MAY_ be given as integer values for `height` and `width` respectively. The image then _SHOULD_ have the service referenced from it.
```
Comment (azaroth42) [inline source/api/presentation/3.0/index.md]: If we were to expand it fully:  It _MUST_ have the `type` property with the value of `"Image"` ? And how far should we abbreviate that?

File: source/api/presentation/3.0/index.md (hunk 1 above)
Comment (tomcrane) [reply source/api/presentation/3.0/index.md]: ...these parts are still considered to be Canvases: their `type` value is the string `Canvas`.

**Comment by tomcrane** [comment]
http://preview.iiif.io/api/1533-prezi-backticks/api/presentation/3.0/

**Pull request merged**